README.md

```diff
@@ -66,9 +85,9 @@ Which of the following statements will work, assuming jQuery is loaded?
 
 Select all that apply:
 ```
-[] `$(".post").css("background", "peachpuff")`
+[x] `$(".post").css("background", "peachpuff")`
 [] `$(".post").innerHTML`
-[] `$(".post").html()`
+[x] `$(".post").html()`
 [] `document.querySelectorAll(".post")[0].innerHTML`
```

Comment:
Yes, those are correct but technically this would work as well:
`[x]  document.querySelectorAll(".post")[0].innerHTML`

Although it's Vanilla JS, it would still work as expected